modules/executive/components/ExecutiveBalance.tsx

```diff
@@ -1,7 +1,7 @@
 import { Flex, Box, Text } from 'theme-ui';
-import Skeleton from 'components/SkeletonThemed';
-import Deposit from 'components/executive/Deposit';
-import Withdraw from 'components/executive/Withdraw';
+import Skeleton from 'modules/app/components/SkeletonThemed';
+import Deposit from '../../mkr/components/Deposit';
+import Withdraw from '../../mkr/components/Withdraw';
```

Comment:
Just a nit-pick but mix of absolute and relative paths triggers me

Reply:
yeah thanks i did some auto imports